docs/guides/integration-mobile-overview.md

```diff
@@ -58,6 +49,108 @@ For details, refer to one of the following guides:
+
+1. How do you want to use the UID2 tokens? See [Paths to Send the Token to the Bidstream](#paths-to-send-the-token-to-the-bidstream).
+
+### Client-Side or Client-Server Integration?
+
+The options for integrating with UID2 using the UID2 mobile SDKs are summarized in the following table. Choose the <Link href="../ref-info/glossary-uid#gl-integration-approaches">integration approach</Link> that's best for you.
+
+| Scenario | Option | Integration Guide |
+| :--- | :--- | :--- |
+| You have access to <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> (email address or phone number) on the client side/within the mobile app, and want to keep changes within your app only. | Client-side integration | [UID2 Client-Side Integration Guide for Mobile](integration-mobile-client-side.md) |
+| You have access to DII on the server side only and can do the necessary development to generate UID2 tokens server-side, or you are using a <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>. | Client-Server Integration | [UID2 Client-Server Integration Guide for Mobile](integration-mobile-client-server.md) |
+
+### Managing the UID2 Token
+
+The following table summarizes the mobile integration options for managing the UID2 token, with links to the documentation for each option.
+
+| UID2 Mobile Implementation Option | Documentation Link |
+| :--- | :---|
+| Prebid.js | Either of the following implementation guides:<ul><li>[UID2 Client-Side Integration Guide for Prebid.js](../guides/integration-prebid-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Prebid.js](../guides/integration-prebid-client-server.md)</li></ul> |
+| UID2 Android SDK | SDK doc: [SDK for Android Reference Guide](../sdks/sdk-ref-android.md)<br/>Implementation Guide:<ul><li>[UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md)</li></ul> |
+| UID2 iOS SDK | SDK doc: [SDK for iOS Reference Guide](../sdks/sdk-ref-ios.md)<br/>Implementation Guide:<ul><li>[UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md)</li></ul> |
+
+[**GWH_SW01 on the above: is the standard Prebid.js implementation guide a correct link? Or should we just link to the optional Prebid.js section in the mobile implementation guide?**]
+
+### Sending the Token to the Bidstream
+
+There are many options for sending your UID2 tokens to the bidstream.
+
+The following table summarizes the options supported by UID2.
+
+| Scenario | Option | Integration Guide |
+| :--- | :--- | :--- |
+| You want to retrieve ads on Google | Secure Signals | [Google Ad Manager Secure Signals Integration Guide](integration-google-ss.md) |
```

Comment:
this is for Web Secure Signal not mobile use case - i dont think we need to mention this

Reply:
Done. thx.